### PR TITLE
Update 10_Versioning.md

### PR DESCRIPTION
**Pull request by manuth**
Fixes typos in `Update 10_Versioning.md`

### CODE REVIEW

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ of `DataObject`. Adding this extension to children of the base class will have u
 
 You can use the `owns` static private property on a DataObject to specify which relationships are ownership relationships. The `owns` property should be defined on the _owner_ DataObject.
 
-For example, let's say you have a `MyPage` page type that displays banners containing an image. Each `MyPage` owns many `Banners`, which in turn owns an `Image`.
+For example, let's say you have a `MyPage` page type that displays banners containing an image. Each `MyPage` owns many `Banners`, which in turn own an `Image`.
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/10_Versioning.md:164]: `owns` is right here.

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md [hunk 2]
```diff
@@ -739,7 +739,7 @@ $record->canViewStage();
 
 #### Customising permissions for a versioned DataObject
 
-Versioned object visibility can be customised in one of the following ways by editing your user code:
+The versioned object visibility can be customised in one of the following ways by editing your user code:
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/10_Versioning.md:742]: I don't think this should be changed. `Versioned` is the name of the feature that versions objects.

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md [hunk 3]
```diff
@@ -842,11 +842,11 @@ public function init()
 
 ### Low level write and publication methods
 
-SilverStripe will usually call these low level methods for you when you. However if you have specialised needs, you may call them directly.
+SilverStripe will usually call these low level methods for you. However if you have specialized needs, you may call them directly.
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/10_Versioning.md:845]: no `z` in specialised

**Review by dhensby — CHANGES_REQUESTED** [review]
a few changes this time :) 🙏

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md (hunk 1 above)
Comment (dhensby) [reply docs/en/02_Developer_Guides/00_Model/10_Versioning.md]: sorry - as in, it *should* be "owns"

Comment (manuth) [reply docs/en/02_Developer_Guides/00_Model/10_Versioning.md]: Oh - sorry for the misunderstanding - I'll change it back right, then

**Review by dhensby — APPROVED** [review]

**Pull request merged**